docs/scenarios_authoring/Enrichers.md

```diff
@@ -41,6 +41,28 @@ DatabaseLookupEnricher is a specialized look-up component; it returns all column
 
 ![alt_text](img/databaseLookupEnricher.png "databaseLookup Enricher")
 
+### DecisionTable enricher
+
+This component allows the user to match input to a pre-defined result. Please be wary that this component works a bit different than a classical decision table([Decision tables in Wikipedia](https://en.wikipedia.org/wiki/Decision_table)). It consists of the fields:
+- Name: the name of the component.
+- Decision Table: a user defined table of type TabularTypedData, discussed below.
+- Match condition: a Boolean SpEL expression that matches input to table rows, discussed below.
+- Output: name of the output variable which will hold data generated by the component.
+- Description: user-defined description of the particular component. 
+
+The component has two main elements: _Decision Table_ which is a user-defined table of values and _Match condition_ - a SpEL expression which Nu will use to match the input to the table values.
+ 
+First we have to define a fixed-value decision table. We will use a simple car mechanic concept - a decision table that helps to define whether a car should be serviced. The blue columns represent our input values and the red one is our decision column.
```

Comment:
We make the distinction that some columns represent input and some decisions, but in case of further description, it isn't important. Because the current implementation of the Decision Table component isn't really a decision table (sic!). I don't think we need to do the distinction. If we do, we should show the example in the wider context. Eg. w the next node in which we could process the decision table node output by taking the first element of the list and extracting the last column (accessing the proper record field) that we interpret as a decision (but at this stage, not in the decision table node).

Reply:
Tbh this has just been added as a remark about flexibility. I think further examples are unnecessary, maybe we should remove the mention of this altogether.